README.md

```diff
@@ -1,73 +1,90 @@
 # ethstorage-sdk
-used for uploading and downloading blobs. The sdk is implemented based on [kzg-wasm](https://github.com/ethereumjs/kzg-wasm/), support the browser environment.
```

Comment:
For a self-contained explanation, I propose adding the following content:
 - What is EthStorage
 - What is the EthStorage contract, and how it looks like as a key-value store
 - What is the purpose of this SDK